tasklist/webapp/src/main/java/io/camunda/tasklist/webapp/api/rest/v1/controllers/TaskController.java

```diff
@@ -385,22 +394,14 @@ public ResponseEntity<TaskResponse> completeTask(
         requireNonNullElse(taskCompleteRequest, new TaskCompleteRequest()).getVariables();
     final var taskSupplier = getTaskSupplier(taskId);
     checkTaskImplementation(taskSupplier);
-    if (!isUserRestrictionEnabled() || hasAccessToTask(taskSupplier)) {
+    if (hasAccessToTask(taskSupplier)) {
       final var completedTask = taskService.completeTask(taskId, variables, true);
       return ResponseEntity.ok(taskMapper.toTaskResponse(completedTask));
     } else {
       throw new ForbiddenActionException(USER_DOES_NOT_HAVE_ACCESS_TO_THIS_TASK_ERROR);
     }
   }
 
-  private boolean isUserRestrictionEnabled() {
-    if (tasklistProperties.getIdentity() != null) {
-      return tasklistProperties.getIdentity().isUserAccessRestrictionsEnabled();
-    } else {
-      return false;
-    }
-  }
```

Comment:
Why is this config check and its usages removed?

Reply:
I removed it as we have a check in the permission class that checks if authorizations are enabled [here](https://github.com/camunda/camunda/blob/main/tasklist/webapp/src/main/java/io/camunda/tasklist/webapp/permission/TasklistPermissionServices.java#L86-L89) which is used in all our permission checks, which I understood to be the same as checking user access restrictions [REF](https://docs.camunda.io/docs/next/components/concepts/access-control/user-task-access-restrictions/#enablingdisabling-user-task-access-restrictions)

Is that not the case?

Reply:
I've now reverted this change